### PR TITLE
Add text for why you want non privileged containers

### PR DESCRIPTION
**Pull request by rhatdan**
Add a description of why you want to run containers without requiring
root.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@cgwalters @alexlarsson PTAL

**Comment by mhausenblas** [comment]
Thanks a lot @rhatdan!

File: README.md [hunk 1]
```diff
@@ -5,7 +5,37 @@ You might not care when launching a single container on your laptop, but in the
 
 ## The problem
 
-TBD.
+### Too many containers require privileges to run.
```
Comment (cgwalters) [inline README.md:8]: It seems strange to not mention e.g. [Kubernetes MustRunAsRange](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/security-context-constraints.md) here.

Comment (mhausenblas) [reply README.md:8]: Yup, SGTM @cgwalters … @rhatdan you plan to add that or shall I merge and add it myself?

**Comment by rhatdan** [comment]
Merge this and add it yourself, or Colin.  We might want to mention this is the default for OpenShift to run all containers without privilege.

**Comment by mhausenblas** [comment]
OK, thanks @rhatdan!

**Pull request merged**